apps/daimo-mobile/src/view/screen/HomeScreen.tsx

```diff
@@ -204,7 +204,8 @@ function HomeScreenInner({ account }: { account: Account }) {
               {!(
                 account.suggestedActions.length > 0 &&
                 netState.status !== "offline" &&
-                isActionVisible
+                isActionVisible &&
+                allComplete
               ) && <Spacer h={allComplete ? 64 : 28} />}
```

Comment:
hmm this is kinda confusing

we already make the spacer taller or shorter depending on `allComplete`

Reply:
so the logic here:
* if we have a suggested action, AND we're online, AND the checklist is done, then don't show any spacer at all
* otherwise, if the checklist is done (but those other conditions don't hold), show a tall spacer
* checklist not done > show a short spacer followed by the "Finish setting up your account" button

Reply:
Going to try simplifying it